From 6baf4b8df94cfc4c1c511e6b39708cd3d27e27d9 Mon Sep 17 00:00:00 2001
From: Wei Yongjun <weiyongjun1@huawei.com>
Date: Mon, 5 Mar 2018 16:21:31 -0800
Subject: [PATCH 012/205] memory: ti-emif-sram: remove redundant dev_err call
 in ti_emif_probe()

This commit comes from:
  git://git.ti.com/processor-sdk/processor-sdk-linux.git

There is a error message within devm_ioremap_resource
already, so remove the dev_err call to avoid redundant
error message.

Signed-off-by: Wei Yongjun <weiyongjun1@huawei.com>
Signed-off-by: Santosh Shilimkar <santosh.shilimkar@oracle.com>
(cherry picked from commit a817e5d82a2dc5daaa0a8a66cfab268fdf2f6bb6)
Signed-off-by: Zumeng Chen <zumeng.chen@windriver.com>
---
 drivers/memory/ti-emif-pm.c |    1 -
 1 file changed, 1 deletion(-)

diff --git a/drivers/memory/ti-emif-pm.c b/drivers/memory/ti-emif-pm.c
index 62a86c4..632651f 100644
--- a/drivers/memory/ti-emif-pm.c
+++ b/drivers/memory/ti-emif-pm.c
@@ -271,7 +271,6 @@ static int ti_emif_probe(struct platform_device *pdev)
 	emif_data->pm_data.ti_emif_base_addr_virt = devm_ioremap_resource(dev,
 									  res);
 	if (IS_ERR(emif_data->pm_data.ti_emif_base_addr_virt)) {
-		dev_err(dev, "could not ioremap emif mem\n");
 		ret = PTR_ERR(emif_data->pm_data.ti_emif_base_addr_virt);
 		return ret;
 	}
-- 
1.7.9.5

